extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -23,7 +23,12 @@ define('ext.wikia.adEngine.lookup.lookupFactory', [
 			response = true;
 			onResponseCallbacks.start();
 
-			adTracker.track(module.name + '/lookup_end', module.getPrices(), 0, 'nodata');
+			if (module.name === 'prebid') {
```

Comment:
Have you thought about making all modules implement `trackAdaptersOnLookupEnd()` and `trackAdaptersSlotState()`?